OpenAI-DotNet/Realtime/RealtimeSessionWebRTC.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using OpenAI.Extensions;
+using System;
+using System.Collections.Concurrent;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using SIPSorcery.Net;
+using System.Linq;
+using System.Text;
+
+namespace OpenAI.Realtime
+{
+    public sealed class RealtimeSessionWebRTC : IDisposable
```

Comment:
seems very similar to the websocket implementation, I wonder if it can be generalized more

Reply:
Yes, it can and it's the main reason the PR is in draft. Lot of reused code. But before refactoring I'd like to know whether adding the SIPSorcery dependency is your prefered approach or not...